packages/chain/package.json

```diff
@@ -16,12 +16,12 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "node --experimental-vm-modules --experimental-wasm-modules --experimental-wasm-threads ./node_modules/jest/bin/jest.js",
+    "test": "node --experimental-vm-modules --experimental-wasm-modules ./node_modules/jest/bin/jest.js",
     "test:watch": "npm run test --watchAll",
     "build": "pnpm run prisma:generate && tsc -p tsconfig.json",
-    "start": "node --experimental-vm-modules --experimental-wasm-modules --experimental-wasm-threads --es-module-specifier-resolution=node ./dist/start.js",
-    "dev:run": "node --loader ts-node/esm --experimental-vm-modules --experimental-wasm-modules --experimental-wasm-threads --es-module-specifier-resolution=node ./src/start.ts start \"./environments/$PROTOKIT_ENV_FOLDER/chain.config.ts\" $1",
-    "dev": "pnpm run prisma:generate && nodemon --watch src -e ts --exec \"pnpm dev:run\"",
+    "start": "node --experimental-vm-modules --experimental-wasm-modules --es-module-specifier-resolution=node ./dist/start.js",
+    "dev:run": "node --loader ts-node/esm --experimental-vm-modules --experimental-wasm-modules --es-module-specifier-resolution=node ./src/start.ts start \"./environments/$PROTOKIT_ENV_FOLDER/chain.config.ts\" $1",
+    "dev": "pnpm run prisma:generate && pnpm run prisma:migrate && nodemon --watch src -e ts --exec \"pnpm dev:run\"",
```

Comment:
Did you test that this was necessary? Did it throw errors if you'd use postgres in a container and then run dev?

Reply:
Yes, if you run just dev straight away it'll complain that it can't find "tables", readme tells you to run migrate by hand, but i think this makes it a lot easier not having to think about it.